benchmarks/benchmark_serving_structured_output.py

```diff
@@ -981,9 +991,9 @@ def main(args: argparse.Namespace):
         "goodput, refer to DistServe paper: https://arxiv.org/pdf/2401.09670 "
         "and the blog: https://hao-ai-lab.github.io/blogs/distserve")
 
-    parser.add_argument("--no-structured-output",
-                        action='store_true',
-                        default=False,
+    parser.add_argument("--structured-output",
+                        action=argparse.BooleanOptionalAction,
+                        default=True,
```

Comment:
any reason this needed to change? seems unrelated?

Reply:
oh I forgot to rever this actually. Should be the same as before.